DESCRIPTION

```diff
@@ -1,6 +1,6 @@
 Package: httr
 Title: Tools for Working with URLs and HTTP
-Version: 1.3.1.9000
+Version: 1.3.2
```

Comment:
Please revert this change - the development version always ends in .9000 (and it's my job to increment when necessary)

Reply:
Done.  I also moved my bullets in NEWS.md under `1.3.1.9000`.